pages/badges/badge-single.jsx

```diff
@@ -133,8 +133,10 @@ var BadgePage = React.createClass({
     if (data.earned) { status = Badge.achieved; }
     if (data.pending) { status = Badge.pending; }
 
-    var reqs = bdata.require_claim_evidence_description;
-    var matched = reqs.match(/((\d+(\.)?)?[^.]+)/g);
+    var evidenceRegex = /((\d+([.\!?])?)?[^.]+)/g;
```

Comment:
regex are giant red warning lights with lots of bells - what are you actually extracting here, and can that be done without regex (because if it can: let's do that instead)